packages/SwingSet/src/lib/id.js

```diff
@@ -2,6 +2,9 @@ import { Nat } from '@endo/nat';
 
 import { Fail } from '@agoric/assert';
 
+const vatIDPattern = /^v[1-9]\d*$/;
```

Comment:
Let's stick with `/^v\d+$/`.. both vatID and deviceID are just a Nat (which allows 0). We happen to start our IDs at 1, but that's not baked into the type.

Reply:
👍

Reply:
`/^v\d+$/` allows e.g. "v007", which I think we don't want: https://github.com/Agoric/agoric-sdk/pull/8040#discussion_r1267504780